packages/firebase_admob/lib/firebase_admob.dart

```diff
@@ -82,6 +82,65 @@ class MobileAdTargetingInfo {
   }
 }
 
+// The types of ad sizes supported for banners.
+enum AdSizeType {
+  WidthAndHeight,
+  SmartBanner,
+}
+
+/// [AdSize] represents the size of a banner ad.
+class AdSize {
```

Comment:
You could just expose const constructors, like `const AdSize.banner(), although it might be simpler to just expose statics, like `AdSize.banner`.

Reply:
I was actually dithering about the most Dart-like way to do this, so I'm glad you brought it up. If having pre-built instances hang out as static constants is allowed, that's my preference.